app/views/devise_otp/credentials/show.html.erb

```diff
@@ -16,7 +16,12 @@
         <%= f.text_field :token, :autocomplete => :off, :autofocus => true, :size => 6, :value => '' %>
     </p>
 
-	<p><%= f.submit I18n.t('submit', {:scope => 'devise.otp.submit_token'}) %></p>
+    <p><%= f.label :trust_device, I18n.t('trust_device', {:scope => 'devise.otp.submit_token'}) %>
+        <%= f.check_box :trust_device, :autocomplete => :off, :checked => false %>
+    </p>
```

Comment:
If you want this checkbox to follow the style of other parts of this gem, you should put a `<br />` after the `label`.  (See https://github.com/ErisExchange/devise-otp/blob/master/app/views/devise_otp/tokens/show.html.erb#L10)

In our application, if we want to change that, we should be providing our own overrides of the views (not just using the gem-provided views).